README.md

```diff
@@ -237,6 +238,31 @@ Each key on the object returned by mapQueriesToProps should be made up of the sa
 
 The `Category` component will also get a prop of `mutations` that will have a key of `postReply`. This key is the method that triggers the mutation and can take custom arguments (e.g. `this.props.mutations.postReply('Apollo and React are really great!')`). These arguments are passed to the method that creates the mutation.
 
+One typical pattern is wanting to refetch a query after a mutation has happened. In this example, `this.props.mutations.postReply`is a method that returns the mutation promise. Since queries pass a `refetch` prop, this can be accomplished like so:
```

Comment:
I think we should remove this readme and just use the docs site, right? Otherwise they will be constantly getting out of sync